src/runtime-rs/crates/agent/src/types.rs

```diff
@@ -9,6 +9,8 @@ use std::convert::TryFrom;
 
 use serde::Deserialize;
 
+pub const DEFAULT_REMOVE_CONTAINER_REQUEST: u32 = 10;
```

Comment:
nit: Name choice
Remove Container Request has two fields, so it would be better to specify the field in the variable name.
consider: `DEFAULT_REMOVE_CONTAINER_REQUEST_TIMEOUT`

Reply:
Done.